package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "tdd-strict-equal",
+  "version": "1.0.0",
+  "description": "week 04 assignment tdd-strict-equal",
+  "main": "index.js",
+  "scripts": {
+    "test": "./node_modules/.bin/mocha",
```

Comment:
not giving any directory to scan for specific files?... we dont want mocha to search all files.

add "./test/**/*.test.js" to the end of line into the quotes